src/components/GNB/Header.tsx

```diff
@@ -90,15 +29,12 @@ export default function Header() {
 
 					<ul className='flex gap-2 sm:gap-4 lg:gap-5'>
 						<li>
-							<Link href={isLoggedIn ? '/mypage' : '/login'}>
-								{isLoggedIn ? '마이페이지' : '로그인'}
-							</Link>
+							{isLoggedIn ? (
+								<ProfileTooltip />
+							) : (
+								<Link href='/login'>로그인</Link>
+							)}
```

Comment:
[p4] 이런 식으로 `hasHydrated` 변수를 활용해 보면 어떨까요?
이건 예시이고, 중첩된 삼항연산자는 가독성에 좋지 않으니 고려하셔서 설계하시면 좋을 거 같습니다ㅎㅎ
```suggestion
							{isLoggedIn ? (
								<ProfileTooltip />
							) : hasHydrated ? (
								<Link href='/login'>로그인</Link>
							) : (
								<div>토큰 가져오는 중...</div>
							)}
```